saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -1050,6 +1074,20 @@ def get_instances_related_to_order(
                         code=OrderBulkCreateErrorCode.INVALID,
                     )
                 )
+        if (
+            not shipping_address
```

Comment:
What if `user` is None? Shouldn't we also raise an exception?

Reply:
We are ignoring it in such scenario. I would like to have the behaviour consistent with other mutation. In case of user is not set we are ignoring the option.

Reply:
I removed support for `OrderBulkCreate` as it was missing checking the address duplication and it's not easy to implement it with good performance. The support will be added if there will be the need for that.